app/platform_stats/rest.py

```diff
@@ -57,7 +57,8 @@ def validate_date_range_is_within_a_financial_year(start_date, end_date):
 
 
 @platform_stats_blueprint.route('usage-for-all-services')
-def get_usage_for_all_services():
+@platform_stats_blueprint.route('data-for-billing-report')
```

Comment:
I kinda feel billing report is this plus the formatting done in the admin app. And this would be data for it 🤷🏼 

As to first question, yes, if you look at admin app PR, you will see that I changed the url there ☺️

Reply:
Fair enough. I don't feel strongly about it, so let's leave it as-is 👍.